.github/workflows/apk.yml

```diff
@@ -52,4 +52,11 @@ jobs:
         with:
           name: "${{ github.event.inputs.name }}-Debug-APK"
           #path: app/build/outputs/apk/release/*.apk
-          path: app/build/outputs/apk/debug/*.apk
\ No newline at end of file
+          path: app/build/outputs/apk/debug/*.apk
+      - name: Create GitHub release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN}}
+        with:
+          tag_name: ${{ github.run_number}}
+          release_name: Release ${{ github.run_number}}
```

Comment:
Please use the provided name from the user input (accessible as `${{ github.event.inputs.name }}`)